### PR TITLE
Altavir/format

### PR DESCRIPTION
**Pull request by altavir**
Add `format` to spec. Fix outdated example.
Implement #12

### CODE REVIEW

File: 5/readme.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
```
Comment (Ingvord) [inline 5/readme.md:1]: same as #16 ?

Comment (altavir) [reply 5/readme.md:1]: I made previous from the master branch. So this one includes the previous one.

Comment (Ingvord) [reply 5/readme.md:1]: Could you please split them? For clarity's sake. Thanks

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ each on a sufficient level for future evolution of Waltz Controls and/or impleme
 
 
 ## Contribution
-
+[]()
```
Comment (Ingvord) [inline README.md:18]: ?

**Review by Ingvord — APPROVED** [review]

**Pull request merged**